client/src/App.jsx

```diff
@@ -1,35 +1,34 @@
 import { useState, useEffect } from 'react';
+//'switch' has been replaced with 'routes'
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+
+import CreateRoseBudThorn from './components/CreateRoseBudThorn';
+import HomePage from './components/HomePage';
+import ListLatestEntrie from './components/ListLatestEntries';
+import RequestBouquet from './components/RequestBouquet';
+import NavBar from './components/NavBar';
+
 import './App.css'
 
 function App() {
 
-  const [backEndMessage, setBackEndMessage] = useState("");
 
-  const callBackEnd = async () => {
-    try{
-      const response = await fetch('http://localhost:8888');
-      if(!response.ok) {
-        throw new Error(`Request failed: ${response.status}`);
-      }
-      const data = await response.json();
-      setBackEndMessage(data);
-    } catch{
-      console.error('An error occured:', error);
-    }
-  }
+  return (
 
-  useEffect(() => {
-    callBackEnd();
-  }, []);
+      <Router>
```

Comment:
people usually put BrowserRouter wrapper in ../main.jsx

Reply:
thank you!  I changed it.